crates/bitwarden-core/src/client/encryption_settings.rs

```diff
@@ -201,6 +205,7 @@ impl EncryptionSettings {
     }
 
     #[cfg(feature = "internal")]
+    #[instrument(err, skip(org_enc_keys, store))]
```

Comment:
We're skipping both arguments, so we can probably use skip_all to match the rest of the functions, right?
```suggestion
    #[instrument(err, skip_all)]
```